context/Dockerfile

```diff
@@ -6,12 +6,15 @@ FROM python:3.6-slim
 RUN apt-get update && apt-get install -y \
   libvips
 RUN pip install \
-  pyvips==2.1.8
+  pyvips==2.1.8 \
+  xmltodict==0.12.0
 
 COPY ometiff_tiler.py .
 
 CMD python ometiff_tiler.py \
     --ometiff_file /input.ome.tif \
     --channel_page_pairs $CHANNEL_PAGE_PAIRS \
+    --server_url $SERVER_URL \
```

Comment:
SERVER_URL is only used for TIFF, I think, and for DZ this would be empty... Is that ok?

Reply:
But we want to support deepzoom still?  We want people to be able to generate Linnarsson data, for example.